docs/Configuration.md

```diff
@@ -114,3 +78,41 @@ spec:
 ```
 
```

Comment:
Shouldn't we add a table of possible and required fields and the default value ?

Reply:
it will may be more cleaner. I will work on it.

Reply:
It looks more clear now.